backend/src/nodes/onnx_nodes.py

```diff
@@ -17,6 +18,11 @@
 from .utils.utils import get_h_w_c, np2nptensor, nptensor2np
 
 
+class TensorOrders(Enum):
```

Comment:
Why did you make this an enum?

Reply:
Because I didn't wanna have magic numbers

Reply:
I meant why you inherit (or whatever magic the `class ClassName(Name)` actually does) from `Enum`. Our other "enum classes" don't do that.

Reply:
Because I googled "python enum" and copied how to do it from there

Reply:
That's what I did too when I first needed to add a dropdown input for Chainner. I quickly removed the `(Enum)` because enums aren't compatible with JSON and I don't want to add `.value` everywhere. That's why I just use static class variables nowadays.

Reply:
I see. I'll change it